### PR TITLE
Maven repair, create readme

### PR DESCRIPTION
**Pull request by eogra7**
This PR includes changes to tidy up the repo by removing compiled files, adding a build script, and beginning documentation,

### CODE REVIEW

File: src/me/delta2force/redditbrowser/RedditBrowserPlugin.java [hunk 1]
```diff
@@ -184,7 +184,7 @@ public void setRoom(Location l, String submissionId) {
 			spawnHologram(bl.clone().add(.5, .5, .5), title);
 		}
 		
-		spawnHologram(bl.clone().add(.5, -.25, .5), "�6"+s.getScore());
+		spawnHologram(bl.clone().add(.5, -.25, .5), "" + (char)(0xfeff00a7) + "6" + s.getScore());
```
Comment (Arnuh) [inline src/me/delta2force/redditbrowser/RedditBrowserPlugin.java:187]: Any reason why this isn't using ChatColor api?

Comment (eogra7) [reply src/me/delta2force/redditbrowser/RedditBrowserPlugin.java:187]: Would you like to implement it? I created this issue #2 for you.

**Comment by darbyjack** [comment]
Hey there, super active developer in the MC community, once this PR is accepted, I'd be happy to go through and clean it up a tad more for you. Yes maven might be annoying if you're not used to it, but I'd be happy to take a few moments and clean it up!

**Comment by delta2force** [comment]
Wow! Thank you so much.

**Comment by delta2force** [comment]
Im kind of worried that no one changed the names of my „I hate maven“ and „IAHTRE“ repos hahaha

**Pull request merged**